charts/rancher-vsphere-cpi/100.6.0+up1.4.2/templates/configmap.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: vsphere-cloud-config
+  labels:
+    vsphere-cpi-infra: config
+    component: {{ .Chart.Name }}-cloud-controller-manager
+  namespace: {{ .Release.Namespace }}
+data:
+  vsphere.yaml: |
+    # Global properties in this section will be used for all specified vCenters unless overriden in VirtualCenter section.
+    {{ with .Values.vCenter }}
+    global:
+      secretName: {{ .credentialsSecret.name | quote }}
+      secretNamespace: {{ $.Release.Namespace | quote }}
+      port: {{ .port }}
+      insecureFlag: {{ .insecureFlag }}
+
+    # vcenter section
+    vcenter:
+      {{ .host | quote }}:
+        server: {{ .host | quote }}
+        datacenters:
+          - {{ .datacenters | quote }}
+    {{- if .labels.generate }}
+
```

Comment:
Super minor but perhaps you might want to move the empty line up to create a logical block for the `vcenter` section?

Reply:
IMO I think we can leave this. The charts works fine and `labels.generate` field does not show up in the configmap because it's a conditional value

Reply:
Valid point, should have handled it as part of https://github.com/rancher/vsphere-charts/pull/45/files :)
Let's leave this "as is".